dali/test/python/test_operator_affine_transforms.py

```diff
@@ -512,3 +516,151 @@ def verify_deprecation(callback):
 def test_transform_translation_deprecation():
     verify_deprecation(lambda : fn.transform_translation(offset=(0, 0)))
     verify_deprecation(lambda : ops.TransformTranslation(offset=(0, 0))())
+
+
+def test_sequences():
+    np_rng = np.random.default_rng(12345)
+    rng = random.Random(42)
+    num_iters = 4
+    max_num_frames = 50
+    max_batch_size = 12
+
+    class TransformsParamsProvider(ParamsProvider):
+        def unfold_output_layout(self, layout):
+            unfolded = super().unfold_output_layout(layout)
+            if unfolded == "**":
+                return ""
+            return unfolded
+
+    def rand_range(limit):
+        return range(rng.randint(1, limit) + 1)
+
+    def mt(desc):
+        return np.float32(np_rng.uniform(-20, 20, (2, 3)))
+
+    def scale(desc):
+        return np.array([rng.randint(0, 5), rng.randint(-50, 20)], dtype=np.float32)
+
+    def shift(desc):
+        return np.array([rng.randint(-100, 200), rng.randint(-50, 20)], dtype=np.float32)
+
+    def shear_angles(desc):
+        return np.array([rng.randint(-90, 90), rng.randint(-90, 90)], dtype=np.float32)
+
+    def angle(desc):
+        return np.array(rng.uniform(-180, 180), dtype=np.float32)
+
+    def per_frame_input(frame_cb):
+        return [[
+            np.array([frame_cb(None) for _ in rand_range(max_num_frames)], dtype=np.float32)
+            for _ in rand_range(max_batch_size)]
+            for _ in range(num_iters)]
+
+    mt_seq_input = per_frame_input(mt)
+
+    test_cases = [
+        (fn.transforms.rotation, {}, TransformsParamsProvider(
+            [ArgCb("angle", angle, True)]), ["cpu"]),
+        (fn.transforms.rotation, {'reverse_order': True}, TransformsParamsProvider(
+            [ArgCb("center", shift, True), ArgCb("angle", angle, False)]), ["cpu"]),
+        (fn.transforms.scale, {}, TransformsParamsProvider(
+            [ArgCb("scale", scale, True), ArgCb("center", shift, False)]), ["cpu"]),
+        (fn.transforms.translation, {}, TransformsParamsProvider(
+            [ArgCb("offset", shift, True)]), ["cpu"]),
+        (fn.transforms.shear, {}, TransformsParamsProvider(
+            [ArgCb("angles", shear_angles, True)]), ["cpu"]),
+        (fn.transforms.shear, {}, TransformsParamsProvider(
+            [ArgCb("shear", shift, True)]), ["cpu"]),
+
+    ]
+    only_with_seq_input_cases = [
+        (fn.transforms.combine, {}, TransformsParamsProvider(
+            [ArgCb(1, mt, True), ArgCb(2, mt, True)]), ["cpu"]),
+        (fn.transforms.combine, {}, TransformsParamsProvider(
+            [ArgCb(1, mt, False)]), ["cpu"]),
+        (fn.transforms.translation, {}, TransformsParamsProvider(
+            [ArgCb("offset", shift, False)]), ["cpu"]),
+    ]
+
+    seq_cases = test_cases + only_with_seq_input_cases
+    yield from sequence_suite_helper(rng, "F", [("F**", mt_seq_input)], seq_cases, num_iters)
```

Comment:
Oh, I guess we just remove `F` in Python test to generate the baseline?

Reply:
Yep.